config/locales/es.yml

```diff
@@ -930,26 +930,7 @@ es:
 
         ¡Gracias!
         <<Preparer.FirstName>> en GetYourRefund.org
-      review_signature_requested: |
-        Hola,
-
-        Su declaración de impuestos para <<TaxReturn.TaxYear>> ya esta lista para su firma electrónica. Por favor revise su declaración para:
-
-        * asegurar que su dirección y sus números de cuenta bancaria esten correctos (Si no estan correctos puede afectar sus números finales y / o puede causar demora para recibir su reembolso.)
-        * revisar que toda la información acerca de sus dependientes este correcta (si aplica a su situación).
-        * asegurar que esten incluidos todos sus ingresos (incluyendo los beneficios de desempleo).
-
-        Si hace falta cambiar esta información o si usted tiene alguna pregunta acerca de su declaración de impuestos, por favor envienos un mensaje de regreso.
-
-        Para aprobar su declaración de impuestos haga click en el enlace más abajo, ingrese los últimos cuatro digitos de su número de seguro social y su número de cliente (Client ID). Puede revisar su declaración y marcar la casilla de verificación que dice que usted aprueba su declaración de impuestos. (Para su información: si usted esta presentando una declaración conjunta con su pareja, es necesario que el o ella de su aprobación también. Por favor comparta este mensaje con su pareja para que de su aprobación.)
-
-        Una vez que usted termine estos pasos, podremos entregar su declaración de impuestos electrónicamente.
-
-        Use este enlace para accesar su declaración y su 8879 (el formulario para que usted de su aprobación):
-        <<Client.LoginLink>>
-
-        Muchas gracias,
-        <<Preparer.FirstName>> de GetYourRefund
+      review_signature_requested: "¡Hola!\n\nSu declaración de impuestos de <TaxReturn.TaxYear>  está lista para que la firme. \n\nPor favor Verifique: \n\n* su dirección postal y números de cuenta bancaria (si esto no es correcto puede afectar sus números finales y/o retrasar su reembolso).\n* toda la información de dependientes \n* asegúrese de que todos los ingresos estén incluidos (incluido el desempleo). \n\nSi necesita cambiar algo en su declaración o si tiene alguna pregunta, porfavor responda a este mensaje y no firme su declaración. \n\nPara aprobar su devolución, haga clic en el enlace a continuación, ingrese los últimos cuatro dígitos de su número de seguro social o su número de cliente, revise su declaración y haga clic en el cuadro que dice que aprueba su declaración.\n\nPara declaraciones presentadas con su cónyuge: su cónyuge también debe revisar y firmar.\n\n Una vez que firme, podemos presentar su declaración electrónicamente. \n\nInicie sesión para acceder a su declaración y aprovar su declaración : <Client.LoginLink>\n\n¡Gracias! \n<Preparer.FirstName>; en Get Your Refund\n"
```

Comment:
The translator wrote `<TaxReturn.TaxYear>` but needs to have written `<<TaxReturn.TaxYear>>`.

The same problem applies to `<<Client.LoginLink>>` and `<<Preparer.FirstName>>`.